File: Ywy/Go/level1/x만큼간격이있는n개의숫자/nNumbersSpacedByX.go
package nNumbersSpacedByX

func Solution(x, n int) []int64 {
	answer := make([]int64, n)
	int64X := int64(x)
	length := int64(n)
	for {
		if length == 0 {
			return answer
		}
		answer[length-1] = int64X * length
		length--
	}
}

File: Ywy/JAVA/src/level1/NNumbersSpacedByX.java
package level1;

public class NNumbersSpacedByX {
	public long[] solution(int x, int n) {
		final long[] answer = new long[n];
		long xx = (long) x;
		while (true) {
			answer[n - 1] = xx * n--;
			if (n == 0) {
				return answer;
			}
		}
	}
}
